src/System.Data.Common/pkg/System.Data.Common.pkgproj

```diff
@@ -3,9 +3,11 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Data.Common.csproj">
-      <SupportedFramework>net451;netcore451;netstandardapp1.5;wpa81</SupportedFramework>
+      <SupportedFramework>net451;netcore451;netstandardapp1.5;wp8;wpa81</SupportedFramework>
     </ProjectReference>
     <ProjectReference Include="..\src\System.Data.Common.builds" />
+    <ExcludeDefaultValidateFramework Include="net45" />
```

Comment:
@ericstj The Packaging.targets in the builds tool tries to validate the package against net45 even though it is not mentioned in the `SupportedFramework `(s)
Is this the right way to exclude validation for net45, or are there other changes needed for this validation exclusion? 

https://github.com/dotnet/buildtools/blob/2f629db77e9c7724a72e81de3bdb5b04ccf1bc62/src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets#L891